dist/aurora.js

```diff
@@ -31,7 +31,7 @@ function isAngularImageDirUser() {
 
 // Detects count of NgOptimizedImage instances with a priority attr
 function getAngularImagePriorityCount() {
-    return document.querySelectorAll('img[ng-img][priority]').length;
+    return document.querySelectorAll('img[ng-img][fetchpriority=high]').length;
```

Comment:
Does it make sense to check for both?

```suggestion
    return document.querySelectorAll('img[ng-img]:is([priority],[fetchpriority=high])').length;
```

Reply:
Sure, I think it works either way (since "priority" should always set "fetchpriority=high"). But may as well be careful!